charts/snapshot-controller/README.md

```diff
@@ -63,26 +64,28 @@ The upgrade procedure can be summarized by the following steps:
 
    If the above commands output any resource, they have to be removed
 
-5. Upgrade the CRDs
+4. Upgrade the CRDs
 
    ```
    kubectl replace -f https://raw.githubusercontent.com/kubernetes-csi/external-snapshotter/v5.0.0/client/config/crd/snapshot.storage.k8s.io_volumesnapshotclasses.yaml
    kubectl replace -f https://raw.githubusercontent.com/kubernetes-csi/external-snapshotter/v5.0.0/client/config/crd/snapshot.storage.k8s.io_volumesnapshots.yaml
    kubectl replace -f https://raw.githubusercontent.com/kubernetes-csi/external-snapshotter/v5.0.0/client/config/crd/snapshot.storage.k8s.io_volumesnapshotcontents.yaml
    ```
 
-6. Upgrade the [snapshot controller](../snapshot-controller) to the latest version:
+5. Upgrade to the latest version:
 
    ```
-   helm upgrade piraeus-charts/snapshot-controller --set image.tag=v5.0.0
+   helm upgrade piraeus-charts/snapshot-controller --set controller.image.tag=v5.0.0 --set webhook.image.tag=v5.0.0
    ```
 
 ## Configuration
 
+### Snapshot controller
 The following options are available:
 
 | Option                      | Usage                                                                                                                  | Default                                                                                            |
 |-----------------------------|------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------|
+| `enabled`                   | Toggle to disable the deployment of the snapshot controller.                                                           | `true`                                                                                             |
```

Comment:
Should probably make it clear somewhere that these values need to be prefixed with `controller` (and `webhook` for the options below)